app/src/features/apiClient/screens/apiClient/components/clientView/components/Collection/CollectionView.tsx

```diff
@@ -136,9 +126,17 @@ export const CollectionView: React.FC<CollectionViewProps> = ({ collectionId })
         <>
           <RQBreadcrumb
             placeholder="New Collection"
-            recordName={collection?.name || "New Collection"}
+            recordName={collectionName}
             onBlur={(newName) => handleCollectionNameChange(newName)}
             autoFocus={location.search.includes("new")}
```

Comment:
Collection breadcrumb will not autofocus because pathname wont contain new now. We should handle this